packages/venia-ui/lib/components/WishlistPage/createWishlist.css

```
@@ -11,6 +60,15 @@
     justify-content: center;
 }
 
-.icon {
-    stroke: rgb(var(--venia-brand-color-1-600));
+.listName {
+    grid-row: 1 / span 1;
+    grid-column: 1 / span 2;
+}
+
+.radioRoot {
+    display: grid;
+    gap: 1rem;
+
+    grid-row: 2 / span 1;
+    grid-column: 1 / span 2;
```

Shouldn't need any of this explicit positioning, the grid would flow this way naturally.

```suggestion
.radioRoot {
    display: grid;
    gap: 1rem;
```